R/utils.R

```diff
@@ -152,11 +152,11 @@ collate_estimates <- function(name, target = "rt"){
   
   # Check a collated file exists
   if(!dir.exists(here::here("subnational", name, "collated", target))){
-    dir.create(here::here("subnational", name, "collated", target))
+    dir.create(here::here("subnational", name, "collated", target), recursive = TRUE)
   }
   
   # Save back to main UK folder
-  data.table::fwrite(df, here::here("subnational", name, "collated", target, paste0(Sys.Date(), ".csv")))
+  data.table::fwrite(df, here::here("subnational", name, "collated", target, paste0('summary_',Sys.Date(), ".csv")))
```

Comment:
not sure this is needed as the expectation is that the collated estimates are summaries?

Reply:
without it the gitignore file hits it as being in the format [0-9]* so ignores the file

Reply:
ah - good spot! All good then.

Reply:
my test run was refusing to show any new files to commit and when I went digging it was there...